thoth/package_extract/image.py

```diff
@@ -178,6 +179,24 @@ def _parse_deb_dependency_line(line_str: str) -> typing.List[tuple]:
     return result
 
 
+def _gather_digests(path: str) -> typing.List[dict]:
+    """Calculate checksum for all files inside image."""
+    digests = []
+    for root, dirs, files in os.walk(path):
+        for file in files:
```

Comment:
It seems to me it might be better to use something like ```file_``` instead of ```file``` because the word ```file``` seem to be reserved in python

Reply:
Indeed, nice catch! :100: